robboat/templates/homepage.html

```diff
@@ -21,9 +21,7 @@ <h1><a href="/"><img src="{% static 'logo.svg' %}" alt="Robb Oat"></a></h1>
       <p><a href="https://github.com/robb-oat/server/issues/2"><img src="{% static 'demo.gif' %}"></a></p>
 
       <p><a href="https://github.com/robb-oat/server/issues/2">Preview Robbʼs work</a>. ☝️</p>
-
-      <p class="cta">👉 <a href="javascript: alert('Not quite yet ...')">“Hire” Robb for $9/mo</a>.</p>
-
+      <p class="cta">👉 <a href="https://github.com/marketplace/robbot-octodad">“Hire” Robb for your org</a>.</p>
```

Comment:
In the ballpark, Robb. In the ballpark.

```suggestion
      <p class="cta">👉 <a href="https://github.com/marketplace/robb-oat">“Hire” Robb for free</a>!</p>
```